repos/system_upgrade/el7toel8/models/repositoriesmap.py

```diff
@@ -0,0 +1,18 @@
+from leapp.models import Model, fields
+from leapp.topics import TransactionTopic 
+
+class RepositoryMap(Model):
```

Comment:
Provide a docstring here, e.g. "Mapping of RHEL 7 repository ID to RHEL 8 repository to determine which RHEL 8 repository to enable for the upgrade."